gnoboard/src/router/path.ts

```diff
@@ -3,6 +3,8 @@ export enum RoutePath {
   ImportPrivateKey = 'ImportPrivateKey',
   GenerateSeedPhrase = 'GenerateSeedPhrase',
   CreatePassword = 'CreatePassword',
+  DeleteAccount = 'DeleteAccount',
+  DeleteConfirm = 'DeleteConfirm',
```

Comment:
The button is renamed to "Remove Account". To avoid confusion, these functions should be renamed to "RemoveAccount" and "RemoveConfirm".